storage_test.go

```diff
@@ -241,6 +242,179 @@ func TestPerformDisplayMultipleRuleDisableOnError(t *testing.T) {
 	}
 }
 
+// TestPerformDisplayMultipleRuleDisableResults checks the basic behaviour of
+// performDisplayMultipleRuleDisable function with results returned.
+func TestPerformDisplayMultipleRuleDisableResults(t *testing.T) {
+	// prepare new mocked connection to database
+	connection, mock, err := sqlmock.New()
+	if err != nil {
+		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+
+	// prepare mocked result for SQL query
+	rows1 := sqlmock.NewRows([]string{"cluster_id", "rule_id", "cnt"})
+	rows1.AddRow("123e4567-e89b-12d3-a456-426614173998", "rule.test|KEY", 1)
+
+	// expected query performed by tested function
+	expectedQuery1 := "select cluster_id, rule_id, count\\(\\*\\) as cnt from cluster_rule_toggle group by cluster_id, rule_id having count\\(\\*\\)>1 order by cnt desc;"
+	mock.ExpectQuery(expectedQuery1).WillReturnRows(rows1)
+
+	// prepare mocked result for SQL query
+	rows2 := sqlmock.NewRows([]string{"org_id"})
+	rows2.AddRow(42)
+
+	// expected query performed by tested function
+	expectedQuery2 := "select org_id from report where cluster = \\$1"
+	mock.ExpectQuery(expectedQuery2).WillReturnRows(rows2)
+	mock.ExpectClose()
+
+	// first query to be performed
+	query1 := `
+                select cluster_id, rule_id, count(*) as cnt
+                  from cluster_rule_toggle
+                 group by cluster_id, rule_id
+                having count(*)>1
+                 order by cnt desc;
+`
+	// call the tested function
+	err = cleaner.PerformDisplayMultipleRuleDisable(connection, nil, query1, "cluster_rule_toggle")
+	if err != nil {
```

Comment:
`assert.Error` or `assert.NoError` is better

Reply:
the tricky part - check if the function does it's job, i.e. what is displayed actually?

Reply:
done both